src/number_theory/arithmetic_function.lean

```diff
@@ -16,6 +16,17 @@ to form the Dirichlet ring.
 
 ## Main Definitions
  * `arithmetic_function α` consists of functions `f : ℕ → α` such that `f 0 = 0`.
+ * An arithmetic function `f` `is_multiplicative` when `x.coprime y → f (x * y) = f x * f y`.
```

Comment:
I guess I should have asked this in the previous PR, but is there a reason that you use `\alpha` for the coefficient ring, instead of `R` or `A`?

Reply:
I just got used to `\alpha` in the `gcd_monoid` PRs, I guess. I'll switch it back to `R`.